pkg/cmd/connector/update/update.go

```diff
@@ -35,85 +41,178 @@ func NewUpdateCommand(f *factory.Factory) *cobra.Command {
 		Short:   f.Localizer.MustLocalize("connector.update.cmd.shortDescription"),
 		Long:    f.Localizer.MustLocalize("connector.update.cmd.longDescription"),
 		Example: f.Localizer.MustLocalize("connector.update.cmd.example"),
-		Hidden:  true,
```

Comment:
We should follow create being hidden. It will be odd to have only update visible :D